src/app/index.ts

```diff
@@ -23,6 +23,8 @@ delete insight.context.tags['ai.device.osVersion'];
 delete insight.context.tags['ai.device.osArchitecture'];
 delete insight.context.tags['ai.device.osPlatform'];
 
+const manifestOnly = 'manifest-only';

```

Comment:
Let's simplify the string "manifest-only" to just be "manifest" as the keyword for the project type.

Reply:
fixed locally